packages/smooth_app/lib/data_models/user_management_provider.dart

```diff
@@ -6,11 +6,7 @@ import 'package:smooth_app/database/dao_secured_string.dart';
 
 class UserManagementProvider with ChangeNotifier {
   static const String _USER_ID = 'user_id';
-  static const String _PASSWORD = 'pasword';
-
-  bool _finishedLoading = false;
-
-  bool get isLoading => !_finishedLoading;
+  static const String _PASSWORD = 'password';
```

Comment:
```suggestion
  static const String _PASSWORD = 'pasword';
```

Reply:
My change will log out some users but I don't see a problem in fixing the naming

Reply:
> My change will log out some users but I don't see a problem in fixing the naming

I don't see a problem in letting an unvisible typo stay either. I'm not going to fight for that, I don't even log in!